client/CMakeLists.txt

```diff
@@ -73,3 +83,9 @@ if(WIN32)
         )
     endif()
 endif()
+
+crashpad_install_target(crashpad_client)
+file(GLOB CRASHPAD_CLIENT_HEADERS "*.h")
```

Comment:
I'll change it to `install(DIRECTORY)`, then I can remove the glob and avoid listing all headers.

Reply:
The installation is only the bare minimum for sentry-native since the compat/tools/... headers are not installed.
But I think it's enough for sentry-native.